### ISSUE TITLE
Discrepancy in SAC on entropy coefficient update

### ISSUE THREAD

**Issue by marioyc** [pfnet/pfrl#177]
Noticed that [here ](https://github.com/pfnet/pfrl/blob/master/pfrl/agents/soft_actor_critic.py#L281) the `log_prob` variable is computed before the udpate of the actor while on SAC's [repo ](https://github.com/rail-berkeley/softlearning/blob/master/softlearning/algorithms/sac.py#L246) it is recomputed after the actor update (the [paper](https://arxiv.org/abs/1812.05905) also mentions in Section 6 that an update is made on both q-function and policy before the update for the entropy coefficient). By any chance have you compared whether this detail makes a difference?

**Comment by muupan** [comment]
You are right, it seems to be a discrepancy from the official implementation. I do not remember whether I made a comparison, maybe not.

**Comment by marioyc** [comment]
I see, no problem, thanks for replying anyways.
